addons/mail/static/src/core/common/search_messages_panel.js

```diff
@@ -18,13 +20,14 @@ export class SearchMessagesPanel extends Component {
     static components = {
         MessageCardList,
         ActionPanel,
+        Activity,
     };
     static props = ["thread", "className?", "closeSearch?", "onClickJump?"];
     static template = "mail.SearchMessagesPanel";
 
     setup() {
         super.setup();
-        this.state = useState({ searchTerm: "", searchedTerm: "" });
+        this.state = useState({ searchTerm: "", searchedTerm: "", activities: undefined });
```

Comment:
It might be easier to handle in code if activities is always an array, empty by default.